stages/01_process.R

```diff
@@ -0,0 +1,42 @@
+library(purrr)
+library(arrow)
+library(fs)
+library(stringr)
+library(dplyr)
+library(tools)
+library(rvest)
+
+download_dir <- "download"
+data_dir <- "brick"
+fs::dir_create(data_dir)
+
+clean_html <- function(html) {
+    raw_text <- rvest::html_text(html)
+    raw_text
+}
+
+save_parquet <- function(file) {
+    print(file)
+    path <- fs::path_ext_remove(file) |>
+        fs::path_ext_set("parquet") |>
+        fs::path_file()
+    file_ext <- file_ext(file)
+    if (file_ext == "fa") {
+		print(file)
+        file.copy(from = file, to = paste0(data_dir, "/", basename(file)))
+    } else if (file_ext == "gff3") {
+        df <- vroom::vroom(file, delim = "\t", comment="#", col_names = c(
+            "seqid", "source", "type", "start", # nolint: indentation_linter.
+            "end", "score", "strand", "phase", "attributes"
+        ))
+        arrow::write_parquet(df, fs::path(data_dir, path))
+    } else if (!grepl("dat|dead", file_ext)) {
```

Comment:
We should skip:

```
download/miRNA.dat
download/miRNA.dead
download/miRNA.diff
```

Since they share file names after removing the suffix, these get overwritten by each other. Just skip them for now. Plus the Python script processes `miRNA.dat`.